autoware.repos

```diff
@@ -16,6 +16,10 @@ repositories:
     type: git
     url: https://github.com/tier4/autoware_auto_msgs.git
     version: tier4/main
+  core/external/ament_cmake:
```

Comment:
@kenji-miyake @xmfcx Is this still necessary to build on arm platform?

Reply:
I guess so.